make glib::Error Send/Sync?

Otherwise it seems rather incompatible with error_chain crate.

I get e.g.:
```
error[E0277]: the trait bound `*mut glib_sys::GError: std::marker::Send` is not satisfied in `errors::Error`
  --> src/errors.rs:9:1
   |
9  | / error_chain! {
10 | |     foreign_links {
11 | |         Alsa(alsa::Error);
12 | |         IO(std::io::Error);
...  |
15 | |     }
16 | | }
   | |_^ `*mut glib_sys::GError` cannot be sent between threads safely
   |
   = help: within `errors::Error`, the trait `std::marker::Send` is not implemented for `*mut glib_sys::GError`
   = note: required because it appears within the type `glib::boxed::AnyBox<glib_sys::GError>`
   = note: required because it appears within the type `glib::boxed::Boxed<glib_sys::GError, glib::error::MemoryManager>`
   = note: required because it appears within the type `glib::Error`
   = note: required because it appears within the type `errors::ErrorKind`
   = note: required because it appears within the type `errors::Error`
   = note: required by `error_chain::ChainedError`
   = note: this error originates in a macro outside of the current crate

error[E0277]: the trait bound `*mut i8: std::marker::Send` is not satisfied in `glib_sys::GError`
  --> src/errors.rs:9:1
   |
9  | / error_chain! {
10 | |     foreign_links {
11 | |         Alsa(alsa::Error);
12 | |         IO(std::io::Error);
...  |
15 | |     }
16 | | }
   | |_^ `*mut i8` cannot be sent between threads safely
   |
   = help: within `glib_sys::GError`, the trait `std::marker::Send` is not implemented for `*mut i8`
   = note: required because it appears within the type `glib_sys::GError`
   = note: required because of the requirements on the impl of `std::marker::Send` for `std::ptr::Unique<glib_sys::GError>`
   = note: required because it appears within the type `std::boxed::Box<glib_sys::GError>`
   = note: required because it appears within the type `glib::boxed::AnyBox<glib_sys::GError>`
   = note: required because it appears within the type `glib::boxed::Boxed<glib_sys::GError, glib::error::MemoryManager>`
   = note: required because it appears within the type `glib::Error`
   = note: required because it appears within the type `errors::ErrorKind`
   = note: required because it appears within the type `errors::Error`
   = note: required by `error_chain::ChainedError`
   = note: this error originates in a macro outside of the current crate
```

--- a/src/error.rs
+++ b/src/error.rs
@@ -25,6 +25,9 @@ glib_wrapper! {
     }
 }
 
+unsafe impl Send for Error {}
+unsafe impl Sync for Error {}
+
 impl Error {
     /// Creates an error with supplied error enum variant and message.
     pub fn new<T: ErrorDomain>(error: T, message: &str) -> Error {

